chapters/competitive-apps.tex

```diff
@@ -0,0 +1,39 @@
+
+Hra se rozprostírá přes 2 obrazovky zařízení a herní postava se ovládá natáčením daného displeje. Pokud například postava vystřelí na jedné obrazovce, střela za chvíli doletí i na obrazovku druhou. Vyvolání příslušných akcí se ovládá určitými kombinacemi dotyků na displej.
+
+UI je velmi jednoduché, avšak přívětivé svými barvami a srozumitelností. Všechny prvky jsou pochopitelné.
+
+\section{Sea Battle 2}
+
+Hra \emph{Sea Battle 2} sice není postavena na kooperaci a komunikaci mezi hráči, ale představuje hezkou reprezentaci klasické deskové hry v moderním pojetí \cite{henrysmithinc_spaceteam}.
+
+Hráči z celého světa mezi sebou na svých zařízeních vedou souboje a vítězstvími postupně postupují v námořních hodnostech. Oproti klasické verzi hra disponuje i pokročilým módem, který obsahuje nejrůznější letadla a pokročilé děla, které obohatí hru o nové strategie.
+
+UI je lazeno do podoby čtverečkovaného papíru s ručně kreslenými prvky, což ale nepůsobí moc přívětivě a tvoří několik problémů, mezi nimiž je například nemožnost přiblížení, což velmi komplikuje některé úkony. 
+
+\section{Keep Talking and Nobody Explodes}
+
+Ve hře \emph{Keep Talking and Nobody Explodes} závisí všechno na správné kooperaci, komunikaci a rychlosti. Tato velmi oceňovaná hra existuje ve verzi pro desktop, konzole, virtuální realitu i mobily \cite{steelcrategamesinc_keep}.
+
+Celá hra stojí okolo problému zneškodnění bomby. Jeden hráč vidí bombu, která je koncipována jako kufřík s několika moduly a časomírou, která se nebezpečně rychle odpočítává. Další hráč, případně skupina, mají k dispozici manuál s instrukcemi. Manuál mohou prohlížet jak v elektronické, tak i ve vytištěné podobě. Veškerá komunikace probíhá pouze slovně.
+
+Hráč s bombou tak nahlásí hráči s manuálem například to, že vidí červené tlačítko s textem \uv{defuse}. Druhý hráč pak v manuálu vyhledá daný modul tlačítka, řídí se instrukcemi a podstatné informace předá prvnímu hráči.
+
+UI je laděné jako pohled na stůl, na kterém je bomba a časomíra, což dodává atmostféru a vtáhne do děje. Světla v místnosti navíc blikají, případně se vypínají, což přidává efekt zejména v situácích, kdy zbývá už jen pár sekund a bomba není vidět. Přestože je grafika jednodušší, samotná bomba je velmi přehledná.
+
+\section{Spaceteam}
+
+Poslední hra \emph{Spaceteam} stojí na rychlosti reakcí a velmi rychlé komunikaci, což shrnuje věta na stránce hry \cite{henrysmithinc_spaceteam} v obchodě Google Play: \uv{Mačkáš rád tlačítka a řveš na své kamarády?}
+
+Herní plocha je plná náhodných tlačítek, přepínačů, posouvátek a dalších věcí, se kterými musí hráči pracovat podle přicházejících \uv{technologicko-nemyslných} příkazů. Zapeklitost komunikace spočívá v tom, že příkazy se netýkají pouze daného zařízení, ale i zařízeních dalších lidí.
```

Comment:
technologicko-nesmyslných?

Reply:
Takhle to mají v popisu hry. Je to trochu neobvyklé použití. Asi to opravdu upravím na něco lepšího.